satpy/resample/ewa.py

```diff
@@ -0,0 +1,11 @@
+"""EWA resamplers."""
+
+from pyresample.ewa import DaskEWAResampler, LegacyDaskEWAResampler
+
+
+def get_ewa_resampler_classes():
+    """Get bucket resampler classes."""
+    return {
+        "ewa": DaskEWAResampler,
+        "ewa_legacy": LegacyDaskEWAResampler,
+    }
```

Comment:
Is the idea that this entire module would go away if/when Resamplers can be loaded directly from a pyresample interface? Or should something like this be replace with a dict/list of strings like I mentioned in the `base.py` comment? That is a series of strings to be imported.

Reply:
The idea was to just kind of follow the structure of the other resample modules and provide a single function to get a dict of the available resamplers to replace the `RESAMPLERS` dict in `__init__.py` but still have the ability to get all of them.